otherlibs/unix/signals.c

```diff
@@ -35,7 +35,7 @@ static void decode_sigset(value vset, sigset_t * set)
 {
   sigemptyset(set);
   while (vset != Val_int(0)) {
-    int sig = caml_convert_signal_number(Int_val(Field(vset, 0)));
+    int sig = caml_convert_signal_number(Int_field(vset, 0));
```

Comment:
This change is correct, but isn't necessary since we don't have read barriers anymore.

Reply:
Fixed in https://github.com/ocaml-multicore/ocaml/commit/d5c2992ecdb2ba9b42f26719492ad9f8b0e8009b